### ISSUE TITLE
Bug - Replace pf-u-screen-reader class in examples/components

### ISSUE THREAD

**Issue by thatblindgeye** [patternfly/patternfly-react#8295]
**Describe the problem**
A clear and concise description of the problem. Which components are affected?
Related to https://github.com/patternfly/patternfly/issues/5195

Currently some examples/components are using an element with class `pf-u-screen-reader`. These should be replaced with `pf-screen-reader` instead. 

This would also apply to components using `accessibleStyles.screenReader` to apply a `className`.
